.dependabot/config.yml

```diff
@@ -8,9 +8,6 @@ update_configs:
       - match:
           dependency_type: "development"
           update_type: "semver:minor"
-    ignored_updates:
-      - match:
-          # Can not be updated to the latest version because it requires ember 3.11+
-          # but this addon supports the last 2 LTS vesions i.e. 3.8 & 3.12
-          dependency_name: "ember-power-select"
-          version_requirement: ">=3"
+    commit_message:
+      prefix: "chore"
+      include_scope: true
```

Comment:
Isn't this the default?

Reply:
No it isnt, if you look at the current dependabot commits they arent prefixed

Reply:
Ah I see, we didn't use conventional commits long enough for dependabot to detect it. In that case, please use this:

```yml
    commit_message:
      prefix: "fix"
      prefix_development: "chore"
      include_scope: true
```

Reply:
Ok changed